apps/src/experiments.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * This module contains logic for tracking various experiments. Experiments
+ * can be enabled/disabled one of two ways
+ * (1) enable: localStorage.set('experiments-experimentName', true)
+ *     disable: localStorage.removeItem('experiments-experimentName')
+ * (2) Add a query param, i.e. http://foo.com?experimentName=true or
+ *     http://foo.com?experimentName=false
+ * The latter approach ends up toggling the localStorage state.
+ * As long as you check isEnabled once for some experiment, data for all
+ * experiments will be loaded from localStorage and the query params will be
+ * parsed.
+ * To support a new experiment, just add it as a key in experimentList
+ */
+var experiments = module.exports;
+
+// List of experiements, and their current enabled/disabled state
+var experimentList = {
+  topInstructions: false,
+  runModeIndicators: false
+};
```

Comment:
It's a little bit of a shame that we need to list the experiments here - it would be nice if adding an experiment was as easy as calling `isEnabled` with an arbitrary string key, much like how localStorage works.